tools/i18n-plugin/pom.xml

```diff
@@ -19,8 +19,8 @@
 
   <properties>
     <maven.core.version>3.6.0</maven.core.version>
-    <maven.plugin.api.version>3.6.0</maven.plugin.api.version>
-    <maven.plugin.annotations.version>3.6.0</maven.plugin.annotations.version>
+    <maven.plugin.api.version>3.9.9</maven.plugin.api.version>
+    <maven.plugin.annotations.version>3.15.1</maven.plugin.annotations.version>
```

Comment:
AFAIK these API versions are usually not set to the latest version as it may also result into devs having to upgrade their Maven version. By not requiring the latest APIs you allow devs more freedom with the Maven version they use. If you really want to enforce everyone (reason?) to always use this Maven version the [enforcer config](https://github.com/openhab/openhab-core/blob/41720b4f0c025f8884064be0b18d48ab656ead3a/pom.xml#L634-L636) should also be updated.

Reply:
Thanks for having a look.

Ok, I have set `plugin.api.version` to 3.6.3 (all before are EOL).
Not sure about `plugin.annotations.version`, I thought it should match version of `plugin-plugin` (which is 3.15.1).

Reply:
As https://github.com/openhab/openhab-core/pull/4664 got merged it now does make sense to update these versions.